test/resources/project/ui5.yaml

```diff
@@ -0,0 +1,10 @@
+---
+specVersion: '2.2'
+type: application
+metadata:
+  name: HTML5Module
+framework:
+  name: OpenUI5
+  version: "1.85.2"
```

Comment:
Is this task version dependent ? Maybe we need to align the version with the runtime version then.